### ISSUE TITLE
interstellar_redist.dll needed on install of 2.0.6

### ISSUE THREAD

**Issue by Videonauth** [net-lisias-ksp/KSP-Recall#29]
Today installed as update KSP Recall and suddendly we had a problem on loading, telling me that `interstellar_redist.dll` is missing. going back to 2.0.5 fixed it for me for now, but maybe this is a mishap in packaging or setup?

**Comment by Lisias** [comment]
Nope. KSP-Recall found some DLL that needs interstellar_redist, but this thing was not loaded by some reason.

Without this DLL, NOTHING from KSPIE will work.

Are you sure you have the interstellar_redist installed?

(in time, this is an issue for ModuleManaget WatchDog, currently packed together with KSP-Recall. Moving this issue to the WatchDog issue tracker)
